lib/config.py

```diff
@@ -144,6 +146,76 @@ def _read_config(self) -> JSON:
 
         return config
 
+    def _read_env_vars(self, config: JSON) -> JSON:
+        # Check Fares
+        check_fares = getenv("AUTO_SOUTHWEST_CHECK_IN_CHECK_FARES")
+        if check_fares:
+            try:
+                config["check_fares"] = is_truthy(check_fares)
+            except ValueError as err:
+                raise ConfigError("Error parsing 'AUTO_SOUTHWEST_CHECK_IN_CHECK_FARES'") from err
+
+        # Notification URL
+        notification_url = getenv("AUTO_SOUTHWEST_CHECK_IN_NOTIFICATION_URL")
+        if notification_url:
+            config.setdefault("notification_urls", [])
+            if isinstance(config["notification_urls"], str):
```

Comment:
I added in a check for an empty string. I purposely wrote all of my logic to happen pre-parse as I felt it fell into the "reading the config" bucket.

Reply:
I forgot that the reason the empty string check is in the `_parse_config` is because that is the default value of notification_urls in the `config.example.json`. However, the env var isn't declared by default and an empty string won't affect the running of the program, so this change can be reverted.